pkg/controller/controller_instance.go

```diff
@@ -1551,9 +1598,15 @@ func (c *controller) processProvisionFailure(instance *v1beta1.ServiceInstance,
 	// requeue this resource.
 	var err error
 	if shouldMitigateOrphan {
-		readyCond := newServiceInstanceReadyCondition(v1beta1.ConditionFalse, startingInstanceOrphanMitigationReason, startingInstanceOrphanMitigationMessage)
+		// Copy original failure reason/message to a new OrphanMitigation condition
 		c.recorder.Event(instance, corev1.EventTypeWarning, readyCond.Reason, readyCond.Message)
-		setServiceInstanceCondition(instance, v1beta1.ServiceInstanceConditionReady, readyCond.Status, readyCond.Reason, readyCond.Message)
+		setServiceInstanceCondition(instance, v1beta1.ServiceInstanceConditionOrphanMitigation,
+			v1beta1.ConditionTrue, readyCond.Reason, readyCond.Message)
+		// Overwrite Ready condition reason/message with reporting on orphan mitigation
+		setServiceInstanceCondition(instance, v1beta1.ServiceInstanceConditionReady,
+			v1beta1.ConditionFalse,
+			startingInstanceOrphanMitigationReason,
+			startingInstanceOrphanMitigationMessage)
 
 		instance.Status.OperationStartTime = nil
 		instance.Status.AsyncOpInProgress = false
```

Comment:
`OrphanMitigationInProgress` is just a flag that means "Orphan mitigation is required" (bad property name, but can't change it since it's part of the API), i.e. this flag doesn't say whether the orphan mitigation has actually started or not.